### PR TITLE
Updated the download page to include DOI access

### PR DESCRIPTION
**Pull request by stephanbracke**
Updated the download page to include info on DOI's. Needed to rename the page again because of references used in technical manual

### CODE REVIEW

**Comment by SimonFlower** [comment]
Is it possible to rename new_data_download -> data_download? I'd been trying to avoid the "new" part of the name, as I don't think it's helpful for people who are new to editing the site.

File: data_download.md [hunk 1]
```diff
@@ -23,4 +25,20 @@ The data portal offers a number of services:
...
+- IRDS 2020: [https://doi.org/10.5880/INTERMAGNET.1991.2020](https://doi.org/10.5880/INTERMAGNET.1991.2020)
+- IRDS 2019: [https://doi.org/10.5880/INTERMAGNET.1991.2019](https://doi.org/10.5880/INTERMAGNET.1991.2019)
+- IRDS 2018: [https://doi.org/10.5880/INTERMAGNET.1991.2018](https://doi.org/10.5880/INTERMAGNET.1991.2018)
+- IRDS 2017: [https://doi.org/10.5880/INTERMAGNET.1991.2017](https://doi.org/10.5880/INTERMAGNET.1991.2017)
+- IRDS 2016: [https://doi.org/10.5880/INTERMAGNET.1991.2016](https://doi.org/10.5880/INTERMAGNET.1991.2016)
+- IRDS 2015: [https://doi.org/10.5880/INTERMAGNET.1991.2015](https://doi.org/10.5880/INTERMAGNET.1991.2015)
+- 2014 publication: [https://doi.org/10.5880/INTERMAGNET.2014](https://doi.org/10.5880/INTERMAGNET.2014)
+- 2013 publication: [https://doi.org/10.5880/INTERMAGNET.2013](https://doi.org/10.5880/INTERMAGNET.2013)
+
+To use the data you will need the BGS software : [Imcdview download](https://geomag.bgs.ac.uk/data_service/intermagnet/home.html) / [installer](https://geomag.bgs.ac.uk/data_service/intermagnet/imcdviewInstaller/install.htm)
```
Comment (SimonFlower) [inline data_download.md:42]: We aren't updating the installer any more (and will probably remove it soon). There's a new Java installer for all BGS Intermagnet software - the place to look is under the heading "Intermagnet Software Installer" on the page https://geomag.bgs.ac.uk/data_service/intermagnet/home.html - I could add an anchor to this heading so you can link directly to it if that helps?

File: data_conditions.md [hunk 2]
```diff
@@ -13,16 +13,7 @@ The data made available through INTERMAGNET are provided for your use and are no
...
-
-- [https://doi.org/10.5880/INTERMAGNET.2013](https://doi.org/10.5880/INTERMAGNET.2013)
-- [https://doi.org/10.5880/INTERMAGNET.2014](https://doi.org/10.5880/INTERMAGNET.2014)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2015](https://doi.org/10.5880/INTERMAGNET.1991.2015)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2016](https://doi.org/10.5880/INTERMAGNET.1991.2016)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2017](https://doi.org/10.5880/INTERMAGNET.1991.2017)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2018](https://doi.org/10.5880/INTERMAGNET.1991.2018)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2019](https://doi.org/10.5880/INTERMAGNET.1991.2019)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2020](https://doi.org/10.5880/INTERMAGNET.1991.2020)
+INTERMAGNET publishes its data in the form of Digital Object Identifiers (DOIs). Definitive INTERMAGNET data publications from 2013 onwards are available as [DOIs](/new_data_download.html#downloading_data_using_dois).
```
Comment (SimonFlower) [inline data_conditions.md]: I think this link needs changing (to /data_download.html) doesn't it?

Comment (stephanbracke) [reply data_conditions.md]: Yes indeed

Comment (stephanbracke) [reply data_conditions.md]: Normally it should be ok now

**Review by SimonFlower — APPROVED** [review]

**Pull request merged**